rcl/src/rcl/subscription.c

```diff
@@ -454,8 +530,13 @@ rcl_subscription_set_content_filter(
     &options->rmw_subscription_content_filter_options);
 
   if (ret != RMW_RET_OK) {
-    RCL_SET_ERROR_MSG(rmw_get_error_string().str);
-    return rcl_convert_rmw_ret_to_rcl_ret(ret);
+    rcl_reset_error();
+    if (!rcl_subscription_common_content_filter_set(
```

Comment:
I believe this behavior is the fallback behavior. if rmw cannot enable CFT, then enable CFT in rcl.

Reply:
Yeah.